server/app/admin.py

```diff
@@ -1,32 +1,85 @@
 from django.contrib import admin
-from .models import Users, Profiles, Tasks, Bids, Payments
+from .models import Tasks, Bids, Payments
+from django.contrib.auth.admin import UserAdmin
+
+from .models import Users
 
 
-# Users Table Interface
 @admin.register(Users)
-class UsersAdmin(admin.ModelAdmin):
-    list_display = ("user_id", "email", "mobile", "created_at",
-                    "updated_at", "last_login", "status", "user_role")
-    list_filter = ("user_role", "status")
+class UserAdminConfig(UserAdmin):
+    model = Users
+    list_display = (
+        "username",
+        "email",
+        "first_name",
+        "last_name",
```

Comment:
If there are first_name and last_name in the db field, then the signup page will need to change accordingly. The signup page may need to add a step to fill in the username as well. @SeiKasahara 
<img width="298" alt="image" src="https://github.com/user-attachments/assets/6dd9706a-7f58-43ff-9bb7-e034f5531445">